husky_components/overlay/main.js

```diff
@@ -36,12 +35,15 @@
  * @params {Function} [options.slides[].closeCallback] @deprecated Use 'cancelCallback' instead
  * @params {Function} [options.slides[].cancelCallback] callback which gets executed after the overlay gets canceled
  * @params {Function} [options.slides[].okCallback] callback which gets executed after the overlay gets submitted
+ * @params {Boolean} [options.slides[].displayHeader] callback which gets executed after the overlay gets submitted
```

Comment:
Does this line make sense? Is `displayHeader` the type of callback described here?